packages/angular/src/utils/version-utils.ts

```diff
@@ -25,3 +26,13 @@ export function getPkgVersionsForAngularMajorVersion(
     ? backwardCompatibleVersions[`angularV${angularMajorVersion}`] ?? versions
     : versions;
 }
+
+export function getInstalledPackageVersionInfo(tree: Tree, pkgName: string) {
+  try {
+    const version = readJson(tree, 'package.json')[pkgName];
+
+    return { major: major(coerce(version)), version };
+  } catch {
+    return null;
+  }
+}
```

Comment:
This is a generator-specific helper and the utilities in this file are meant to be used for everything inside the plugin, not just generators. Could you move this to `packages/angular/src/generators/utils/angular-version-utils.ts`? We can rename that file to just `versions-utils.ts` so it's more generic.

Reply:
ah yes indeed, no problem.